cmd/server.go

```diff
@@ -72,6 +72,7 @@ const (
 	SlackTokenFlag             = "slack-token"
 	SSLCertFileFlag            = "ssl-cert-file"
 	SSLKeyFileFlag             = "ssl-key-file"
+	StatusName                 = "status-name"
```

Comment:
```suggestion
	PRStatusName                 = "pr-status-name"
```

I think adding PR will save some confusion as to what status we're talking about.